lib/components/RunFrameForCli/LeftHeader.tsx

```diff
@@ -145,6 +129,14 @@ export const RunframeCliLeftHeader = (props: {
     } as RequestToSaveSnippetEvent)
   }
 
+  const { isExporting, startExport } = useExportHandler({
+    requestToExportSentAt,
+    setRequestToExportSentAt,
+    recentEvents,
+    setIsError,
+    pushEvent,
```

Comment:
this hook should probably just use the store for pushEvent, it needlessly makes this file longer to have it as a param